src/electron.f90

```diff
@@ -25,7 +25,7 @@ module electron_module
   use wannier_module, only: wannier
   use crystal_module, only: crystal, calculate_wavevectors_full
   use symmetry_module, only: symmetry, find_irred_wedge, create_fbz2ibz_map
-  use delta, only: form_tetrahedra_3d, fill_tetrahedra_3d, form_triangles, &
+  use delta, only: form_tetrahedra_3d, fermi_window_adjusted_fill_tetrahedra_3d, form_triangles, &
```

Comment:
Please do not remove the preexisting `fill_tetrahedra_3d`. We are not yet sure if your new implementation is correct.